### ISSUE TITLE
This module should be able to install a version other than the latest from the repo.

### ISSUE THREAD

**Issue by pong-takepart** [deric/puppet-mesos#78]
Presently, the 1.0.1-2.0.93.centos65 mesos RPM is corrupted in the mesosphere respositories; it is also the latest in the repository. This module appears to be able to [only install the latest from that repo](https://github.com/deric/puppet-mesos/blob/master/manifests/install.pp#L38) so the puppet run breaks.

Yum has the capability to install a previous version in the repo, but this module is unable to take advantage of that:

```
yum install mesos-1.0.0-2.0.89.centos65
```

The example above installs a version previous to the latest that is not corrupted:

```
[root@master1 ~]# yum list --showduplicates mesos
Loaded plugins: fastestmirror, security
Loading mirror speeds from cached hostfile
 * base: mirror.n5tech.com
 * epel: linux.mirrors.es.net
 * extras: mirror.n5tech.com
 * updates: repos.lax.quadranet.com
Available Packages
mesos.x86_64                 0.19.0-1.1.centos64                      mesosphere
mesos.x86_64                 0.19.1-1.0.centos64                      mesosphere
mesos.x86_64                 0.20.0-1.0.centos64                      mesosphere
mesos.x86_64                 0.20.1-1.0.centos64                      mesosphere
mesos.x86_64                 0.21.0-1.0.centos65                      mesosphere
mesos.x86_64                 0.21.1-1.1.centos65                      mesosphere
mesos.x86_64                 0.21.2-0.2.63.centos65                   mesosphere
mesos.x86_64                 0.22.0-1.0.centos65                      mesosphere
mesos.x86_64                 0.22.1-1.0.centos65                      mesosphere
mesos.x86_64                 0.22.2-0.2.62.centos65                   mesosphere
mesos.x86_64                 0.23.0-1.0.centos65                      mesosphere
mesos.x86_64                 0.23.1-0.2.61.centos65                   mesosphere
mesos.x86_64                 0.24.0-1.0.27.centos65                   mesosphere
mesos.x86_64                 0.24.1-0.2.35.centos65                   mesosphere
mesos.x86_64                 0.24.2-2.0.17.centos65                   mesosphere
mesos.x86_64                 0.25.0-0.2.70.centos65                   mesosphere
mesos.x86_64                 0.25.1-2.0.21.centos65                   mesosphere
mesos.x86_64                 0.26.0-0.2.145.centos65                  mesosphere
mesos.x86_64                 0.26.1-2.0.19.centos65                   mesosphere
mesos.x86_64                 0.26.2-2.0.60.centos65                   mesosphere
mesos.x86_64                 0.27.0-0.2.190.centos65                  mesosphere
mesos.x86_64                 0.27.1-2.0.226.centos65                  mesosphere
mesos.x86_64                 0.27.2-2.0.15.centos65                   mesosphere
mesos.x86_64                 0.28.0-2.0.16.centos65                   mesosphere
mesos.x86_64                 0.28.1-2.0.20.centos65                   mesosphere
mesos.x86_64                 0.28.2-2.0.27.centos65                   mesosphere
mesos.x86_64                 1.0.0-2.0.89.centos65                    mesosphere
mesos.x86_64                 1.0.1-2.0.93.centos65                    mesosphere
```

In conclusion, this module would be able to take advantage of this capability by adding a 'package_name' paramater to mesos::install such that the package name becomes the name used by [the package declaration on line 38](https://github.com/deric/puppet-mesos/blob/master/manifests/install.pp#L38).

**Comment by pong-takepart** [comment]
Sorry. I'm ignorant. Closing.
